### PR TITLE
Bug fixes and adding readme.md, changelog.md

### PR DESCRIPTION
**Pull request by Arsfiqball**
Ketika saya buka ``http://localhost:8000/po-admin/``
Kolom password terbuka seperti biasanya.
Tapi ketika saya buka ``http://localhost:8000/po-admin``
Kolom password tidak terbuka sama sekali.
Setelah saya cek di browser console, ternyata javascript mengirimkan url yang salah.
Jadi solusi dari saya membuat variabel global untuk javascript (bisa dilihat pada commit).

note: saya menggunakan command ``php -S localhost:8000`` untuk menjalakan server.

😉

### CODE REVIEW

**Comment by Arsfiqball** [comment]
Ups...
sepertinya saya lupa mematikan auto converter tab ke spasi...

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+- Penambahan fitur Speech Recognition pada component post sehingga menulis tidak perlu lagi mengetikan keyboard tetapi cukup dengan berbicara pada microphone saja (Only Google Chrome).
+- Penambahan user registrasi menjadikan PopojiCMS menjadi web multiuser yang dinamis sesuai dengan hak akses tiap user.
+- Penambahan component user akses sehingga secara default hak akses terdiri dari admin, user dan member, yang nantinya bisa ditambahkan sendiri hak akses sesuai kebutuhan end user.
+- Penambahan component user role dimana component ini dapat mengatur secara dinamis hak akses masing-masing content yang ada di halaman administrator sesuai dengan user akses tertentu.
+- Perbaikan .htaccess dimana telah diatur slug url baru yang lebih SEO Friendly lagi, hal ini untuk menanggapi permintaan dari kebanyakan end user yang menginginkan slug url yang lebih baik.
+- Perbaikan 2 theme default mengikuti slug url baru serta perbaikan fitur pagging yang lebih dinamis lagi.
+- Perbaikan pada title, meta descriptions, meta keywords untuk hasil SEO yang lebih baik lagi.
+
+### Build Version 1.2.4 (7 Juni 2014) :
```
Comment (jayvdb) [inline changelog.md]: In other dates, you have used a leading `0` like `07 Juni 2014`.

File: changelog.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+- Perbaikan pada menu engine, dimana sekarang sudah bisa support multi class sampai 3 class ke bawah.
+- Perbaikan pada .htaccess baik pada frond-end ataupun pada back-end.
+- Perbaikan setiap inputan pada frond-end tidak lagi menggunakan ajax tetapi melalui pemprosesan php murni, hal ini untuk mempermudah developing theme nantinya.
+- Penambahan fitur pagging dimana sekarang bisa dipilih untuk pagging berupa kombinasi number atau hanya tombol prev dan next saja.
+- Perbaikan penulisan coding pada komponen galeri dan user.
+- Perbaikan pada komponen post dimana sekarang untuk user yang tidak memiliki hak akses admin hanya bisa melihat dan mengedit konten yang ditulis oleh dirinya sendiri.
+
+### Build Version 1.2.3 (1 April 2014) :
```
Comment (jayvdb) [inline changelog.md]: lagi

File: po-admin/index.php [hunk 3]
```diff
@@ -34,50 +34,57 @@
...
+    <script type="text/javascript" src="../<?=DIR_INC;?>/js/patternlock/patternLock.js"></script>
+    <script type="text/javascript" src="../<?=DIR_INC;?>/js/login-core.js"></script>
 </body>
 </html>
 <?php
 } else {
-	header('location:admin.php?mod=home');
+    header('location:admin.php?mod=home');
 }
 ?>
```
Comment (jayvdb) [inline po-admin/index.php]: maybe add EOL at EOF?

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# PopojiCMS
+Free Content Management System - Indonesia
+Contact : info@popojicms.org
+
+## Cara instalasi PopojiCMS
+
```
Comment (jayvdb) [inline readme.md]: 0. Download from https://github.com/PopojiCMS/PopojiCMS/releases/latest ?

File: po-admin/index.php (hunk 3 above)
Comment (Arsfiqball) [inline po-admin/index.php]: Actually my SublimeText code editor configured to add EOL at EOF automatically. But too many files like that (has no EOL at EOF). I think it is not good to change every file by just opening it. So I rolled back and turned it off.

File: readme.md (hunk 4 above)
Comment (Arsfiqball) [inline readme.md]: Oh yah, I forgot to include other option (download and clone from github). The download mentioned means download button in PopojiCMS official website.

File: po-admin/index.php (hunk 3 above)
Comment (Arsfiqball) [inline po-admin/index.php]: > Ups...
> sepertinya saya lupa mematikan auto converter tab ke spasi...

I put standarized php coding style (PSR-4) configuration in my sublimetext. And I forgot to turn it all off. Maintainer seems like doesn't follow it at all.
I will be more carefull next time.

**Comment by jayvdb** [comment]
@Arsfiqball , is an updated patch coming ?

**Pull request merged**